### PR TITLE
EO-628 Health Score Gauge

### PR DESCRIPTION
**Pull request by jonambas**
This PR isn't opened against master. Will need to do some cleanup with EO-632 when that is ready.

### What Changed
- Added new components to support these mocks: https://sparkpost.invisionapp.com/share/H8R0LV8E5NA#/352294734_Health_Dashboard

### How To Test
- Point your local upstreams to staging
- Log into appteam or enable the `feature_signals_v2` UI option
- Navigate to the new Health Score page from the main side nav

### To Do
- [ ] Address Feedback
- [ ] Lift the GET health score request up to the page component from the summary table
  - May hold on this to avoid ballooning this PR

### CODE REVIEW

File: src/helpers/signals.js [hunk 1]
```diff
@@ -28,3 +29,28 @@ export const getFriendlyTitle = ({ prefix, facet, facetId, subaccountId }) => {
...
+
+export const getDoD = (current, before) => {
+  if (_.isNil(current) || _.isNil(before)) {
+    return null;
+  }
+
+  return ((current - before) / before * 100);
+};
+
+export const getCaretProps = (value, reverse) => {
```
Comment (jonambas) [inline src/helpers/signals.js]: Might just move this into the MetricDisplay component

File: src/selectors/signals.js [hunk 2]
```diff
@@ -308,7 +309,8 @@ export const selectHealthScoreOverviewData = createSelector(
         normalizedHistory.reduce((total, { health_score }) => total + health_score, 0) / normalizedHistory.length,
         1
       ),
-      WoW: _.isNil(WoW) ? null : roundToPlaces(WoW * 100, 0)
+      WoW: _.isNil(WoW) ? null : roundToPlaces(WoW * 100, 0),
+      current_DoD: getDoD(_.last(filledHistory).health_score, filledHistory[filledHistory.length - 2].health_score)
```
Comment (PatrickSison) [inline src/selectors/signals.js]: This key is `current_DoD` but in CurrentHealthGuage.js, it's expecting the key to be `DoD`

**Comment by PatrickSison** [comment]
Will the EO-632 put a grid on the 2 sections for that and this ticket? On mobile view, the 2 panels could potentially stack vertically

File: package-lock.json [hunk 3]
```diff
@@ -5237,7 +5237,7 @@
     },
```
[the remark this answers is not in the document]
Comment (jonambas) [reply package-lock.json]: Nope, will fix

File: src/selectors/signals.js (hunk 2 above)
Comment (jonambas) [reply src/selectors/signals.js]: Oh good catch, one of the downsides of using sparse datasets

**Pull request merged**